packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -178,7 +178,7 @@ class TextPainter {
   /// This and [textDirection] must be non-null before you call [layout].
   ///
   /// The [InlineSpan] this provides is in the form of a tree that may contain
-  /// multiple instances of [TextSpan]s and [WidgetSpan]s. To obtain a plaintext
+  /// multiple instances of [TextSpan]s and [WidgetSpan]s. To obtain a plain text
```

Comment:
"plaintext" is (slightly) more common than "plain text"

https://books.google.com/ngrams/graph?content=plaintext%2C+plain+text&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cplaintext%3B%2Cc0%3B.t1%3B%2Cplain%20text%3B%2Cc0